.github/workflows/prod-build-push-container.yml

```diff
@@ -43,7 +45,9 @@ jobs:
         run: |
           docker build -t base \
           --build-arg PRODUCTION_ENV=true \
-          --build-arg GITHUB_SHA_ARG=$GITHUB_SHA .
+          --build-arg GITHUB_SHA_ARG=$GITHUB_SHA \
+          --build-arg GOOGLE_CLIENT_SECRET=$GOOGLE_CLIENT_SECRET \
+          --build-arg GOOGLE_CLIENT_ID=$GOOGLE_CLIENT_ID .
```

Comment:
Just a note that build args are visible to anyone with the Docker image using the `docker history` command.

A few ways to work around it are:
1. Lookup the secrets on ECS task startup directly from SecretsManager; or
2. Use [Docker secrets](https://pythonspeed.com/articles/docker-build-secrets/).

Reply:
Unfortunately these secrets are included in built files and thus negating the effect of using Docker secrets or the like